cmd/kube-proxy/app/server.go

```diff
@@ -88,7 +95,8 @@ func checkKnownProxyMode(proxyMode string) bool {
 // Options contains everything necessary to create and run a proxy server.
 type Options struct {
 	// ConfigFile is the location of the proxy server's configuration file.
-	ConfigFile string
+	ConfigFile  string
+	WriteConfig string
```

Comment:
It is currently a path name. I debated having `--write-config` be a `bool`, which would mean you'd do `--config=foo.yaml --write-config`, but that's not what we did in OpenShift, so I went this route instead. What do you think makes the most sense?